scripts/manage-versions.js

```diff
@@ -158,9 +159,10 @@ async function runAdd() {
   await fetchAndSaveDiffsForVersion(selectedVersion);
 
   // Update versions file
-  const updatedVersions = currentVersions
-    .concat(selectedVersion)
-    .sort((a, b) => (parseFloat(a) < parseFloat(b) ? 1 : -1));
+  const updatedVersions = orderBy(currentVersions.concat(selectedVersion), (version) => {
+    const [major, minor] = version.split('.');
+    return major * 10 + minor;
```

Comment:
works until we have subversions with 3 digits ;)

Reply:
we should use sort still but change the comparison, the comparison will first compare the major verison, if the same then compare the subversion.

Reply:
hmmm I just realize `const [a, b, c] = [1, 2]` will just pass with `c` being `undefined`. Was afraid that'll fail... will update :)

Reply:
we aren't using patch versions, just a.b so non issue